src/helix-ui/elements/HXDrawerElement.less

```diff
@@ -0,0 +1,166 @@
+@import "./HXElement";
+@import (reference) "components/buttons";
+
+#hxClose {
+  #Button.reset();
+  color: var(--hxClose-color, @gray-700);
+  font-size: 1rem;
+  height: 2.75rem; // 44px for a11y
```

Comment:
Height and width had to be adjusted to meet minimum touch target size for accessibility. 

@araiford, we'll need to make sure that the redlines are updated to match.

Reply:
Got it.  Will add that adjustment to the color palette story for the Drawer